src/IO/Exporter/Exporter.cpp

```diff
@@ -0,0 +1,52 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "IO/Exporter/Exporter.hpp"
+
+#include "IO/Exporter/PointwiseInterpolator.hpp"
+#include "Utilities/GenerateInstantiations.hpp"
+
+namespace spectre::Exporter {
+
+template <size_t Dim>
+std::vector<std::vector<double>> interpolate_to_points(
+    const std::variant<std::vector<std::string>, std::string>&
+        volume_files_or_glob,
+    const std::string& subfile_name, const ObservationVariant& observation,
+    const std::vector<std::string>& tensor_components,
+    const std::array<std::vector<double>, Dim>& target_points,
+    const bool extrapolate_into_excisions,
+    const std::optional<size_t> num_threads) {
+  tnsr::I<DataVector, Dim, Frame::Inertial> target_points_dv{};
+  for (size_t d = 0; d < Dim; ++d) {
+    target_points_dv.get(d).set_data_ref(
+        const_cast<double*>(gsl::at(target_points, d).data()),
```

Comment:
Thow some NOLINTs in there or use `make_const_view`

Reply:
I don't think we have `make_const_view` for `double*`, so I added the nolint